packages/asset-service/src/service/descriptions.json

```diff
@@ -1,22 +1,184 @@
+    "fr": "",
+    "ru": "",
+    "es": "",
+    "id": "",
+    "ko": "",
+    "pt": "",
+    "zh": "",
+  }
+  "eip155:1/erc20:0xb9446c4ef5ebe66268da6700d26f96273de3d571": {
+    "en": "This token represents a liquidity pool on Curve, a leading decentralized exchange. Holders earn fees from users trading in the pool, and can also deposit the LP token on [ShapeShift](/defi/earn) to earn yield. This liquidity pool contains [CRV](/assets/eip155:1/erc20:0xd533a949740bb3306d119cc777fa900ba034cd52) and [ETH](/assets/eip155:1/slip44:60). To acquire Curve CRV-ETH, deposit [CRV](/assets/eip155:1/erc20:0xd533a949740bb3306d119cc777fa900ba034cd52) and/or [ETH](/assets/eip155:1/slip44:60) on [Curve](https://curve.fi/crveth/deposit). Please be aware that as liquidity pools are composed of differently-priced assets, they are subject to impermanent loss.",
+    "fr": "",
+    "ru": "",
+    "es": "",
+    "id": "",
+    "ko": "",
+    "pt": "",
+    "zh": "",
+  }
+  "eip155:1/erc20:0x3a283d9c08e8b55966afb64c515f5143cf907611": {
+    "en": "This token represents a liquidity pool on Curve, a leading decentralized exchange. Holders earn fees from users trading in the pool, and can also deposit the LP token on [ShapeShift](/defi/earn) to earn yield. This liquidity pool contains [CRV](/assets/eip155:1/erc20:0xd533a949740bb3306d119cc777fa900ba034cd52) and [ETH](/assets/eip155:1/slip44:60). To acquire Curve CRV-ETH, deposit [CRV](/assets/eip155:1/erc20:0xd533a949740bb3306d119cc777fa900ba034cd52) and/or [ETH](/assets/eip155:1/slip44:60) on [Curve](https://curve.fi/crveth/deposit). Please be aware that as liquidity pools are composed of differently-priced assets, they are subject to impermanent loss.",
+    "fr": "",
+    "ru": "",
+    "es": "",
+    "id": "",
+    "ko": "",
+    "pt": "",
+    "zh": "",
+  }
   "eip155:1/erc20:0xdc49108ce5c57bc3408c3a5e95f3d864ec386ed3": "FOXy is a rebasing ERC-20 Token, which you can exchange 1:1 with FOX Tokens, that offer both yield and profit redistribution from the ShapeShift DAO. Get more out of your FOX without losing your governance power.",
   "cosmos:cosmoshub-4/slip44:118": "Cosmos (ATOM) is a cryptocurrency that powers an ecosystem of blockchains designed to scale and interoperate with each other. The Cosmos vision aligns with the ShapeShift vision of a multi chain future, and enables a network of blockchains to be able to communicate with each other in a decentralized manner. Cosmos is a proof-of-stake chain and allows holders to stake their ATOM to earn rewards using ShapeShift.",
   "cosmos:osmosis-1/ibc:27394FB092D2ECCD56123C74F36E4C1F926001CEADA9CA97EA622B25F41E5EB2": "Cosmos (ATOM) on Osmosis is a proof of stake cryptocurrency that powers an ecosystem of blockchains designed to interoperate and communicate with each other in a decentralized manner on the Osmosis blockchain, an alternative decentralized [IBC](https://tendermint.com/ibc/) blockchain."

```

Comment:
These will need to be the same structure as the others.

Also, this structure change will break the existing tests in AssetService.test.ts. The mocked structure for descriptions.json will need to be updated to the new structure.

Reply:
Done. Should be done now.

Reply:
Is it? There is no mocks update in this diff